devenv/dev-dashboards/datasource_tests_elasticsearch_v2.json

```diff
@@ -586,14 +573,88 @@
       "title": "ES Log query",
       "transform": "json",
       "type": "table"
+    },
+    {
+      "circleMaxSize": 30,
+      "circleMinSize": 2,
+      "colors": ["rgba(245, 54, 54, 0.9)", "rgba(237, 129, 40, 0.89)", "rgba(50, 172, 45, 0.97)"],
+      "datasource": "gdev-elasticsearch-v2-metrics",
+      "decimals": 0,
+      "esMetric": "Count",
+      "gridPos": {
+        "h": 12,
+        "w": 24,
+        "x": 0,
+        "y": 27
+      },
+      "hideEmpty": false,
+      "hideZero": false,
+      "id": 8,
+      "initialZoom": 1,
+      "links": [],
+      "locationData": "countries_3letter",
```

Comment:
Looks like the query is for geohashes but the location data is set to match on 3 letter country names. When looking at the map, one of the geohashes happen to match with Serbia but the other 7 results did not match and are not shown on the map.

The query also has a group by time which is maybe a bit weird for the Worldmap as it a sort of Singlestat panel.

To match on Geohashes the options should look like this:

![image](https://user-images.githubusercontent.com/434655/55347950-4f233800-54b6-11e9-8189-2770a50f458e.png)

Reply:
Good catch! Have updated and forced push